docs/ABOUT.md

```diff
@@ -12,6 +12,7 @@ You should learn JavaScript because:
 
 * It's easy to learn.
 * It's versatile in the sense that it's multi-paradigm supporting procedural, event based, object oriented and functional programming.
+* It can be used for the frontend and backend.
```

Comment:
hey @rchavarria , done the changes you suggest and Im excited to contribute my favorite interview question to the list of available problems!! Thanks for helping